pokemongo_bot/event_handlers/discord_handler.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from pokemongo_bot.event_manager import EventHandler
+from pokemongo_bot.base_dir import _base_dir
+import json
+import os
+import time
+import discord_simple
+import thread
+import re
+from pokemongo_bot.datastore import Datastore
+import pprint
+
+
+DEBUG_ON = False
+
+class FileIOException(Exception):
+    pass
+
+class DiscordClass:
+
+    update_id = None
+
+
+    def __init__(self, bot, master, pokemons, config):
+        self.bot = bot
+        self.pokemons = pokemons
+        self._dbot = None
+        self.config = config
+
+    def sendMessage(self, to=None, parse_mode='Markdown', text=None):
+        self._dbot.send_message(to, text)
+
+    def connect(self):
+        self._dbot = discord_simple.Bot(self.bot.config.discord_token,on_message=self.on_message)
+
+    def _get_player_stats(self):
+        web_inventory = os.path.join(_base_dir, "web", "inventory-%s.json" % self.bot.config.username)
+        try:
+            with open(web_inventory, "r") as infile:
+                json_inventory = json.load(infile)
+        except ValueError as exception:
+            self.bot.logger.info('[x] Error while opening inventory file for read: %s' % exception)
+            json_inventory = []
+        except:
+            raise FileIOException("Unexpected error reading from {}".format(web_inventory))
+        return next((x["inventory_item_data"]["player_stats"]
+                     for x in json_inventory
+                     if x.get("inventory_item_data", {}).get("player_stats", {})),
+                    None)
+
+    def send_player_stats_to_chat(self, chat_id):
+        stats = self._get_player_stats()
+        if stats:
+            with self.bot.database as conn:
+                cur = conn.cursor()
+                cur.execute("SELECT DISTINCT COUNT(encounter_id) FROM catch_log WHERE dated >= datetime('now','-1 day')")
+                catch_day = cur.fetchone()[0]
+                cur.execute("SELECT DISTINCT COUNT(pokestop) FROM pokestop_log WHERE dated >= datetime('now','-1 day')")
+                ps_day = cur.fetchone()[0]
+                res = (
+                    "*"+self.bot.config.username+"*",
+                    "_Level:_ "+str(stats["level"]),
+                    "_XP:_ "+str(stats["experience"])+"/"+str(stats["next_level_xp"]),
+                    "_Pokemons Captured:_ "+str(stats["pokemons_captured"])+" ("+str(catch_day)+" _last 24h_)",
+                    "_Poke Stop Visits:_ "+str(stats["poke_stop_visits"])+" ("+str(ps_day)+" _last 24h_)",
+                    "_KM Walked:_ "+str("%.2f" % stats["km_walked"])
+                )
+            self.sendMessage(to=chat_id, parse_mode='Markdown', text="\n".join(res))
+        else:
+            self.sendMessage(to=chat_id, parse_mode='Markdown', text="Stats not loaded yet\n")
```

Comment:
In future can rename "TelegramTask" config keys to some like "alerts" and add key: agents: ["telegram", "discord", "email"]